docs/clients.md

```diff
@@ -69,6 +69,43 @@ Therefore these two rules may help when choosing what to use:
 * When opening a client within a service, use local clients.  Usually you'll want to open one client per worker.
 * When opening a client from outside a service, such as from inside an actor or some other general use case, use a future client.
 
+## Failure Management
+ 
+### Client Behavior 
+ 
+A Client handles makes a request to an external system.  If any failure occurs
+(example: connection closed, external system isn't responding) the response is 
+sent back as a failure and NOT retried.  The default behavior is to continue
+trying subsequent requests.  If failfast is enabled then all subsequent 
+requests that are queued up in the ServiceClient are removed.  
+
+### Retrying Requests
+
+To enable retries, the LoadBalancingClient (LBC) should be used.  The LBC takes
```

Comment:
I think it's worth pointing out that a LBC can open multiple connections, so it can be used for more than just reties....like actual load balancing.